controllers/operands/kubevirt_test.go

```diff
@@ -2524,6 +2523,104 @@ Version: 1.2.3`)
 			}))
 		})
 
+		Context("Tune KubeVirt rateLimiters", func() {
+
+			var hco *hcov1beta1.HyperConverged
+
+			BeforeEach(func() {
+				hco = commonTestUtils.NewHco()
+			})
+
+			It("Should be empty by default", func() {
+				kv, err := NewKubeVirt(nil, nil, hco)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(hco.Spec.TuningPolicy).To(BeEmpty())
+				Expect(kv.Spec.Configuration.APIConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.ControllerConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.WebhookConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.HandlerConfiguration).To(BeNil())
+
+			})
+
+			It("Should be empty if configmap is not present", func() {
+				hco.Spec.TuningPolicy = hcov1beta1.HyperConvergedTuningPolicyConfigMap
+				cl := commonTestUtils.InitClient([]runtime.Object{})
+
+				kv, err := NewKubeVirt(context.TODO(), cl, hco)
+				Expect(err).To(HaveOccurred())
+				Expect(kv).To(BeNil())
+
+			})
+
+			It("Should be empty if the configmap is present but the parameters are wrong", func() {
+
+				hco.Spec.TuningPolicy = hcov1beta1.HyperConvergedTuningPolicyConfigMap
+				//burst is missing
+				configMapData := make(map[string]string, 0)
+				configMapData[qps] = "100"
+
+				configMap := corev1.ConfigMap{
+					TypeMeta: metav1.TypeMeta{
+						Kind:       "ConfigMap",
+						APIVersion: "v1",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      string(hcov1beta1.HyperConvergedTuningPolicyConfigMap),
+						Namespace: getNamespace(hco.Namespace, []string{}),
+					},
+					Data: configMapData,
+				}
+
+				resources := []runtime.Object{&configMap}
+
+				cl := commonTestUtils.InitClient(resources)
+
+				kv, err := NewKubeVirt(context.TODO(), cl, hco)
+
+				Expect(err).To(HaveOccurred())
+				Expect(kv).To(BeNil())
+
+			})
+
+			It("Should create the fields and populate them when requested", func() {
+				hco.Spec.TuningPolicy = hcov1beta1.HyperConvergedTuningPolicyConfigMap
+				//burst is missing
+				configMapData := make(map[string]string, 0)
+				configMapData[qps] = "100"
+				configMapData[burst] = "200"
+
```

Comment:
maybe this would be more clear?

```suggestion
				configMapData := map[string]string {
				    qps: "100",
				    burst: "200",
				}
```

Reply:
I agree. Thanks for the suggestion.